statsmodels/survey/survey_table.py

```diff
@@ -0,0 +1,188 @@
+        # stderr to the total calculated for each group
+        self.var_dict = {}
+        for ind in self.df_group.indices.values():
+            # make vector of zeros
+            group_weights = np.zeros(self._m)
+            # except at the indices in a particular group
+            group_weights[ind] = self.design.weights[ind]
+            group_design = ss.SurveyDesign(self.design.strat,
+                                           self.design.clust,
+                                           np.ones(self._m))
+
+            d = np.vstack([group_weights, self.design.weights]).T
+            self.var_dict[tuple(ind)] = np.square(ss.SurveyRatio(group_design,
+                                                  data=d,
+                                                  cov_method='linearized',
+                                                  center_by='stratum').stderr)
+            group_var = np.asarray(list(self.var_dict.values()))
+            group_var = group_var.reshape(len(group_var), )
+            self.group_var = group_var
+        return group_var
+
+    def _delta(self, cell_prop=True):
+        # Constructs 'delta', whose eigenvalues are used for corrections
+        # of the LRT and Pearson statistic
+
+        # Diagonal matrix of the cell proportions
+        D_inv = np.linalg.inv(np.diag(self._cell_prop.values.flatten()))
+
+        # v_hat may or may not be in order bc _group_variance() used dict
+        # indices to grab the observations, and dictionaries aren't ordered
+        v_hat = np.diag(self._group_variance())
+        self.stderr = np.sqrt(v_hat)
+
+        # get v_srs using cell proportions or proportions under the null
+        if cell_prop:
+            v_srs = np.outer(self._col_prop, self._col_prop)
+            np.fill_diagonal(v_srs, self._col_prop * (1-self._col_prop))
+            v_srs /= self._m
+        else:
+            v_srs = np.outer(self._null, self._null) / self._m
+            np.fill_diagonal(v_srs, self._col_prop * (1-self._col_prop))
+            v_srs /= self._m
+
+        # get constrast matrix
+        b = self._contrast_matrix()
+        self.b = b.copy()
+
+        # only making these attributes for testing until result matches STATA
+        self.delta_numer = np.dot(b.T, D_inv).dot(v_hat).dot(D_inv).dot(b)
+        self.delta_denom = np.dot(b.T, D_inv).dot(v_srs).dot(D_inv).dot(b)
+        self.delta_denom = np.linalg.inv(self.delta_denom)
+        self._delta_est = np.dot(self.delta_denom, self.delta_numer)
+
+    def _contrast_matrix(self):
+        # Builds a contrast matrix, full-rank matrix orthogonal to [1|mat]
+        mat = self._main_effects_mat()
+        u, s, vt = np.linalg.svd(mat, 0)
+        b = u[:, s > 1e-12]
+
+        qm = np.eye(b.shape[0]) - np.dot(b, b.T)
+        u, s, vt = np.linalg.svd(qm, 0)
+        b = u[:, s > 1e-12]
+        return b
+
+    def _main_effects_mat(self):
+        # Builds a matrix that contains “main effects” for the rows and columns
```

Comment:
use plain quotes `"`  
The quotes here are not ASCII, and there is no encoding declaration on top of the file
We use only ASCII except for proper names that have non-ASCII characters